config/checkstyle_checks.xml

```diff
@@ -134,7 +134,7 @@
   </module>
   <module name="RegexpSingleline">
     <property name="id" value="assertThatShouldBeOnSeparateLine"/>
-    <property name="format" value="assertWithMessage\(.*\).that\("/>
+    <property name="format" value="^((?!\*).)*assertWithMessage\(.*\).that\("/>
```

Comment:
Don't think its been asked, please explain this change.

Reply:
We are basically correcting the pattern to only match actual method calls and not methods calls present in javadocs.

```java
/**
 * void method() {
 *   assertWithMessage("ex").that(1).isEqualTo(1); // ok, identifierPattern is different
 * }
 */ 
```
It used to match the method call in the javadoc too, but now it doesn't.

Reply:
Then why not use `^[^*]*assertWithMessage\(.*\).that\("/>`?

Reply:
Done, don't know why I used that regex back then.